CHANGELOG.md

```diff
@@ -15,18 +17,10 @@
   the targets `sql.clickhouse`, `sql.duckdb` and `sql.postgres` to work around a
   serious bug of compiler. (@eitsupi, #3788)
```

Comment:
```suggestion
 bug of compiler. We'll attempt to re-enable the feature in an upcoming release. (@eitsupi, #3788)
```

Reply:
I wonder we can merge #3792 before 0.10.1.